library/ecp.c

```diff
@@ -2781,6 +3140,11 @@ int mbedtls_ecp_check_privkey( const mbedtls_ecp_group *grp,
             return( 0 );
     }
 #endif /* ECP_SHORTWEIERSTRASS */
+#if defined(ECP_EDWARDS)
+    /* FIXME: add a check for Edwards */
```

Comment:
Note: I'll need to check the references in more depths, but it might be that we have nothing to check here, except the size of the key (no more than x bytes).